molo/yourwords/migrations/0006_create_your_words_index_pages.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def create_competition_index(apps, schema_editor):
+    from molo.core.models import Main
+    from molo.yourwords.models import YourWordsCompetition, YourWordsCompetitionIndexPage
+    main = Main.objects.all().first()
+
+    if main:
+        competition_index = YourWordsCompetitionIndexPage(title='Your words competitions', slug='Your-words-competitions')
+        main.add_child(instance=competition_index)
+        competition_index.save_revision().publish()
+
+        # Move existing competitions
```

Comment:
Same here, we should just create the index, but leave the content where it is. Will have to add instructions to the release notes.